extensions/ql-vscode/src/vscode-tests/cli-integration/remote-queries/variant-analysis-monitor.test.ts

```diff
@@ -96,12 +98,31 @@ describe('Variant Analysis Monitor', async function() {
         expect(result.variantAnalysis?.status).to.equal(VariantAnalysisStatus.Failed);
         expect(result.variantAnalysis?.failureReason).to.equal(processFailureReason(mockFailedApiResponse.failure_reason as VariantAnalysisFailureReason));
       });
+
+      it('should emit `onVariantAnalysisChange`', async () => {
+        const spy = sandbox.spy();
+        variantAnalysisMonitor.onVariantAnalysisChange(spy);
+
+        await variantAnalysisMonitor.monitorVariantAnalysis(variantAnalysis, cancellationTokenSource.token);
+
+        expect(spy).to.have.been.calledOnce;
+      });
     });
 
-    describe('when the variant analysis completes', async () => {
+    describe('when the variant analysis is being monitored', async () => {
```

Comment:
So the case above this: `when the variant analysis fails` also occurs when the variant analysis is being monitored, but it fails instead of finishing. Perhaps `when the variant analysis is in progress` is a good way to describe this group of tests? 

Alternatively we could get rid of this extra `describe` since it doesn't have a `beforeEach` setup.

Reply:
Good point! Changed 💅🏽